src/features/events/components/EventOverviewCard/EventParticipantsCard.tsx

```diff
@@ -0,0 +1,221 @@
+                    model.setReqParticipants(newReqParticipants);
+                  }
+                }}
+              >
+                <Paper elevation={3} variant="elevation">
+                  <Box mt={1} p={2}>
+                    <TextField
+                      helperText={messages.eventParticipantsCard.reqParticipantsHelperText()}
+                      label={messages.eventParticipantsCard.reqParticipantsLabel()}
+                      onChange={(ev) => {
+                        const val = ev.target.value;
+
+                        if (val == '') {
+                          setNewReqParticipants(null);
+                          return;
+                        }
+
+                        const intVal = parseInt(val);
+                        if (!isNaN(intVal) && intVal.toString() == val) {
+                          setNewReqParticipants(intVal);
+                        }
+                      }}
+                      onKeyDown={(ev) => {
+                        if (ev.key === 'Enter') {
+                          setAnchorEl(null);
+                          if (newReqParticipants != null) {
+                            model.setReqParticipants(newReqParticipants);
+                          }
+                        } else if (ev.key === 'Escape') {
+                          setAnchorEl(null);
+                          setNewReqParticipants(reqParticipants);
+                        }
+                      }}
+                      value={
+                        newReqParticipants === null ? '' : newReqParticipants
+                      }
+                      variant="outlined"
+                    />
+                  </Box>
+                </Paper>
+              </ClickAwayListener>
+            </Popper>
+          </Box>
+        }
+      >
+        <Divider />
+        <Box
+          display="flex"
+          flexDirection="column"
+          justifyContent="space-between"
+        >
+          <Box
+            alignItems="center"
+            display="flex"
+            justifyContent="space-between"
+            marginY={1}
+          >
+            <Typography
+              color={theme.palette.text.secondary}
+              component="h6"
+              variant="subtitle1"
+            >
+              {messages.eventParticipantsCard.pending()}
+            </Typography>
+            <Typography>
+              {Math.max(reqParticipants - availParticipants, 0)}
+            </Typography>
+          </Box>
+          <Box
+            alignItems="center"
+            display="flex"
+            justifyContent="space-between"
+            marginBottom={1}
+          >
+            <Typography
+              color={theme.palette.text.secondary}
+              component="h6"
+              variant="subtitle1"
+            >
+              {messages.eventParticipantsCard.booked()}
+            </Typography>
+            <Typography>{`${availParticipants}/${availParticipants}`}</Typography>
```

Comment:
I'm curious here about the numbers displayed here. My interpretation is that this will always show the same number on both sides, and I what I expect is the right side number being all participants, and the left number being those that are confirmed or notified?

Reply:
As in comment on `pending`, this requires additional data, which I have dealt with in #1199 which is still not merged. Let's leave this as is, and solve it once both are merged.